packages/app-mobile/components/ExtendedWebView.tsx

```diff
@@ -150,6 +153,7 @@ const ExtendedWebView = (props: Props, ref: Ref<WebViewControl>) => {
 			onMessage={props.onMessage}
 			onError={props.onError}
 			onLoadEnd={props.onLoadEnd}
+			decelerationRate='normal'
```

Comment:
This fixes the "scroll feels non-native" issue we were previously experiencing when using the `WebView` top-level scroll.

This should also allow removing some iOS-specific viewer CSS that was added to work around this issue.